examples/mcp-client/wrangler.jsonc

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "mcp-client",
+  "main": "src/server.ts",
+  "compatibility_date": "2025-03-14",
+  "compatibility_flags": ["nodejs_compat"],
+  "vars": {
```

Comment:
probably not necessary to have this in wrangler.jsonc, just make it a regular var in the source code?